### PR TITLE
Fix type and case sensitivity of time-ingestion partitioned table pseudo columns

### PR DESCRIPTION
**Pull request by jphalip**

### CODE REVIEW

File: connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/avro/AvroRecordReader.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ private Object serializeRow(GenericRecord record) {
     List<Schema.Field> fields = actualSchema.getFields();
     Object[] row = new Object[columnNames.size()];
     for (Schema.Field field : fields) {
-      int colIndex = columnNames.indexOf(field.name());
+      int colIndex = columnNames.indexOf(field.name().toLowerCase());
```
Comment (yigress) [inline connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/avro/AvroRecordReader.java:71]: can we add test case for external table bq column name (not just pesudo column) have upper case?

Comment (jphalip) [reply connector/src/main/java/com/google/cloud/hive/bigquery/connector/input/avro/AvroRecordReader.java:71]: Good point. I made this small change to address that: https://github.com/GoogleCloudDataproc/hive-bigquery-connector/pull/98/commits/2b0e33d76ec5913c2b28ded199ef5d662f844152

**Comment by yigress** [comment]
tests not run because of spotless check

**Comment by jphalip** [comment]
Ok I just re-pushed with spotless

**Review by yigress — APPROVED** [review]

**Pull request merged**